### PR TITLE
Admin forms

### PR DESCRIPTION
**Pull request by cstillitano**
- Remove status for new decks and cards #32 
- Add status and tier to single deck summary #31 
- Style guiding quote radio buttons #30

### CODE REVIEW

File: app/views/cards/_card_form.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   </ul>
   <div class="tab-content">
     <!-- FILL_IN_BLANK -->
-    <div class="<%= !@card.card_type || @card.card_type === "fill_in_blank" ? "tab-pane active" : "tab-pane" %>" id="fill_in_blank">
+    <div class="<%= !@card.id || @card.card_type === "fill_in_blank" ? "tab-pane active" : "tab-pane" %>" id="fill_in_blank">
```
Comment (findandrew) [inline app/views/cards/_card_form.html.erb]: `@card.new_record?` is a bit more expressive than `!@card.id`, you just need two equals `==`

File: app/views/cards/_card_form.html.erb [hunk 2]
```diff
@@ -47,11 +47,13 @@
...
-          <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
-        </div>
-        <%= f.submit @card.card_type ? 'Update' : 'Create', class: 'btn btn-brand' %>
+        <% if @card.id %>
+          <div class="form-group">
+            <%= f.label :status %>
+            <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
+          </div>
+        <% end %>
+        <%= f.submit @card.id ? 'Update' : 'Create', class: 'btn btn-brand' %>
```
Comment (findandrew) [inline app/views/cards/_card_form.html.erb]: same: `@card.new_record?` is more expressive. There's also `.persisted?` if you don't want to swap the order of Update and Create

File: app/views/cards/_card_form.html.erb [hunk 3]
```diff
@@ -47,11 +47,13 @@
           <%= f.text_area :content, class: 'form-control', rows: '5', placeholder: 'Fill in the blank prompt ...' %>
           <p class="help-block">Add blanks -> #blank here#'</p>
         </div>
-        <div class="form-group">
-          <%= f.label :status %>
-          <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
-        </div>
-        <%= f.submit @card.card_type ? 'Update' : 'Create', class: 'btn btn-brand' %>
+        <% if @card.id %>
```
Comment (findandrew) [inline app/views/cards/_card_form.html.erb]: `@card.persisted?`

File: app/views/cards/_card_form.html.erb [hunk 4]
```diff
@@ -87,11 +89,13 @@
           <label>Answer</label>
           <textarea class="form-control" rows="4" placeholder="Flashcard answer ..."></textarea>
         </div>
-        <div class="form-group">
-          <%= f.label :status %>
-          <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
-        </div>
-        <%= f.submit @card.card_type ? 'Update' : 'Create', class: 'btn btn-brand' %>
+        <% if @card.id %>
```
Comment (findandrew) [inline app/views/cards/_card_form.html.erb]: `@card.persisted?`

File: app/views/cards/_card_form.html.erb [hunk 5]
```diff
@@ -87,11 +89,13 @@
...
+          <div class="form-group">
+            <%= f.label :status %>
+            <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
+          </div>
+        <% end %>
+        <%= f.submit @card.id ? 'Update' : 'Create', class: 'btn btn-brand' %>
```
Comment (findandrew) [inline app/views/cards/_card_form.html.erb]: `@card.persisted?` and for the rest of `@card.id` and `@deck.id` (im not gonna spam you with the same comment a dozen times

Comment (cstillitano) [inline app/views/cards/_card_form.html.erb]: gotta keep it DRY

**Comment by cstillitano** [comment]
@findandrew updated to check for `persisted?`

**Pull request merged**